extensions/oidc/deployment/src/main/java/io/quarkus/oidc/deployment/devservices/keycloak/KeycloakOpenApiHandler.java

```diff
@@ -0,0 +1,48 @@
+package io.quarkus.oidc.deployment.devservices.keycloak;
+
+import io.quarkus.devconsole.runtime.spi.DevConsolePostHandler;
+import io.vertx.core.MultiMap;
+import io.vertx.ext.web.RoutingContext;
+import io.vertx.mutiny.core.buffer.Buffer;
+import io.vertx.mutiny.ext.web.client.HttpResponse;
+import io.vertx.mutiny.ext.web.client.WebClient;
+
+public class KeycloakOpenApiHandler extends DevConsolePostHandler {
+    @Override
+    protected void handlePostAsync(RoutingContext event, MultiMap form) throws Exception {
+        WebClient client = KeycloakDevServicesUtils.createWebClient();
+        try {
+            
+            String openapi = client.getAbs("http://locahost:8080/q/openapi").send().onItem()
```

Comment:
if `/q/openapi` is available then the openapi extension is added and this voids ".e having a swagger-ui/openapi dependency should really not be a prerequisite for using this Dev UI." 

I still think we should just have a link on the dev ui page that goes to swagger ui with the logged in state...